sdk/metricsadvisor/ai-metrics-advisor/src/models.ts

```diff
@@ -185,6 +185,19 @@ export type DataFeedGranularity =
       granularityType: "Custom";
       customGranularityValue: number;
     };
+/**
+ * The flattened response to a REST call.
+ * Contains the underlying HttpOperationResponse as well as
+ * the merged properties of the parsedBody, parsedHeaders, etc.
+ */
+export interface RestResponse {
```

Comment:
Can you point where this is used?

Reply:
It was used in delete methods, e.g., https://github.com/Azure/azure-sdk-for-js/blob/14099039a8009d6d9687daf65d22a998e3ad7920/sdk/metricsadvisor/ai-metrics-advisor/src/metricsAdvisorAdministrationClient.ts#L523

Reply:
https://github.com/Azure/azure-sdk-for-js/pull/18736/files#r771766141